java/query/src/main/java/sleeper/query/executor/QueryExecutor.java

```diff
@@ -245,4 +258,21 @@ protected List<String> getFiles(Partition partition) {
         }
         return files;
     }
+
+    public boolean cacheRefreshRequired() {
+        boolean result = cacheExpireTime.isBefore(LocalDateTime.now());
```

Comment:
Instant should be more generally reliable than LocalDateTime because it uses a fixed time zone regardless of where it runs (UTC).